share/spice/grunt/grunt.js

```diff
@@ -0,0 +1,101 @@
+(function (env) {
+    "use strict";
+    
+    // From http://stackoverflow.com/a/34064434/1546577
+    // This is used to decode entities that is often found in the descriptions.
+    function htmlDecode(input) {
+        var doc = new DOMParser().parseFromString(input, "text/html");
```

Comment:
Looks like this isn't supported by IE9 :frowning: 

What entities are expected? Can we do it manually?

We need a standard function for this, even something as brute force as http://underscorejs.org/docs/underscore.html#section-160